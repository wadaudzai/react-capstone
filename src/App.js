import React from 'react';
import { Route, Routes } from 'react-router';
import { BrowserRouter } from 'react-router-dom';
import Coins from './components/Coins';
import Navbar from './components/Navbar';
import CoinsInfo from './components/CoinsInfo';

const App = () => (
  <BrowserRouter>
    <Navbar />
    <Routes>
      <Route path="/Coins" element={<Coins />} />
      <Route path="/CoinsInfo" element={<CoinsInfo />} />
    </Routes>
  </BrowserRouter>
);

export default App;
