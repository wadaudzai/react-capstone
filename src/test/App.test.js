import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import App from '../App';
import store from '../redux/configureStore';

it('Test App', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <App />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
