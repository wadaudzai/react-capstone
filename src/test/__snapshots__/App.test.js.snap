// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App 1`] = `
Array [
  <nav
    className="navbar"
  >
    <ul>
      <li
        className="content-holder"
      >
        <a
          aria-current="page"
          className="holder active"
          href="/"
          onClick={[Function]}
        >
          <svg
            className="back-icon"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 448 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M257.5 445.1l-22.2 22.2c-9.4 9.4-24.6 9.4-33.9 0L7 273c-9.4-9.4-9.4-24.6 0-33.9L201.4 44.7c9.4-9.4 24.6-9.4 33.9 0l22.2 22.2c9.5 9.5 9.3 25-.4 34.3L136.6 216H424c13.3 0 24 10.7 24 24v32c0 13.3-10.7 24-24 24H136.6l120.5 114.8c9.8 9.3 10 24.8.4 34.3z"
            />
          </svg>
          <div
            className="nav-holder"
          >
            <img
              alt="CoinImage"
              className="nav-image"
              src="headlogo.png"
            />
          </div>
          <svg
            className="back-icon"
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            strokeWidth="0"
            style={
              Object {
                "color": undefined,
              }
            }
            viewBox="0 0 512 512"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M255.545 8c-66.269.119-126.438 26.233-170.86 68.685L48.971 40.971C33.851 25.851 8 36.559 8 57.941V192c0 13.255 10.745 24 24 24h134.059c21.382 0 32.09-25.851 16.971-40.971l-41.75-41.75c30.864-28.899 70.801-44.907 113.23-45.273 92.398-.798 170.283 73.977 169.484 169.442C423.236 348.009 349.816 424 256 424c-41.127 0-79.997-14.678-110.63-41.556-4.743-4.161-11.906-3.908-16.368.553L89.34 422.659c-4.872 4.872-4.631 12.815.482 17.433C133.798 479.813 192.074 504 256 504c136.966 0 247.999-111.033 248-247.998C504.001 119.193 392.354 7.755 255.545 8z"
            />
          </svg>
        </a>
      </li>
    </ul>
  </nav>,
  <div
    className="app-holder"
  >
    <div
      className="form-holder"
    >
      <input
        className="form"
        onChange={[Function]}
        placeholder="Search a cryptocurrency"
        type="text"
      />
    </div>
    <div
      className="main-holder"
    />
  </div>,
]
`;
